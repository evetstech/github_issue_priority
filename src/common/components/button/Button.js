import React from 'react';
import './Button.scss';

const Button = (props) => {
  const { onClick, disabled, text } = props;

  return (
    <button onClick={onClick} disabled={disabled} className='btn-rounded'>
      {text}
    </button>
  );
};

export default Button; 