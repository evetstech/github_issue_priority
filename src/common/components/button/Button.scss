.btn-rounded {
  min-height: 30px;
  height: 5%;
  min-width: 100px;
  width: 10%;
  background-color: white;
  border: 1px solid black;
  user-select: none;
}