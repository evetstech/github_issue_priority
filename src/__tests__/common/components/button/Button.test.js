import React from 'react';
import { mount } from 'enzyme';
import Button from '../../../../common/components/button/Button';
import renderer from 'react-test-renderer';

describe('button', () => {
  it('should mount', () => {
    const wrapper = mount(<Button/>);

    expect(wrapper.length).toBeTruthy();
  });

  it('renders correctly', () => {
    const tree = renderer
      .create(<Button text={'hi'}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});