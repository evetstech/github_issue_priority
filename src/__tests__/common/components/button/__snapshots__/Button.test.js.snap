// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button renders correctly 1`] = `
<button
  className="btn-rounded"
>
  hi
</button>
`;
